_includes/codepens/checklist/index.js

```diff
@@ -0,0 +1,6 @@
+tinymce.init({
+  selector: "textarea",
```

Comment:
This should be "textarea#checklist", otherwise it could try and override other areas on the page.